cypress_shared/pages/temporary-accommodation/manage/premisesShow.ts

```diff
@@ -5,7 +5,7 @@ import paths from '../../../../server/paths/temporary-accommodation/manage'
 
 export default class PremisesShowPage extends Page {
   constructor(private readonly premises: Premises) {
-    super('Manage a property')
+    super('View a property')
```

Comment:
The commit message for this change is the wrong way round: 'Change "manage a property" to "view a property" is correct

Reply:
👍